### DISCUSSION TITLE
进程默认只监听ipv6了，这个得怎么设置才能保证所有服务只监听ipv4端口

### DISCUSSION THREAD

**Discussion by lifekevin21** [apache/doris#21590] [A - General / Q&A]
[root@localhost ~]# docker ps -a
CONTAINER ID   IMAGE                                COMMAND                 CREATED         STATUS         PORTS     NAMES
5352c83f5ef7   apache/doris:2.0.0_alpha-be-x86_64   "bash entry_point.sh"   2 minutes ago   Up 2 minutes             root-be-1
b8848cd14f5d   apache/doris:2.0.0_alpha-fe-x86_64   "bash init_fe.sh"       2 minutes ago   Up 2 minutes             root-fe-1

[root@localhost ~]# netstat -tunlp
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      1057/sshd
tcp        0      0 127.0.0.1:25            0.0.0.0:*               LISTEN      1350/master
tcp6       0      0 192.168.10.49:9010      :::*                    LISTEN      9679/java
tcp6       0      0 :::22                   :::*                    LISTEN      1057/sshd
tcp6       0      0 ::1:25                  :::*                    LISTEN      1350/master
tcp6       0      0 :::9020                 :::*                    LISTEN      9679/java
tcp6       0      0 :::8030                 :::*                    LISTEN      9679/java
tcp6       0      0 :::9030                 :::*                    LISTEN      9679/java
udp        0      0 0.0.0.0:68              0.0.0.0:*                           8939/dhclient

使用官网提供的docker-compose.yaml来部署的，但是只监听了ipv6，这个要怎么搞？

**Comment by yagagagaga** [comment]
配置 priority_networks 参数
